blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,225 @@
+
+Krystian Hebel, Sergii Dmytruk, and Yaroslav Kurlaev have made major contributions
+to developing and maintaining support for Power platforms. Their changes include
+[emulation improvements](https://review.coreboot.org/c/coreboot/+/67061),
+and fixing ongoing issues related to these platforms.
+
+#### Platform security (TPM, TXT)
+
+Our team also worked on [TPM](https://3mdeb.com/shop/modules/tpm-2-0/) and
+[TXT](https://blog.3mdeb.com/2022/2022-03-17-optiplex-txt/) solutions that
+could support the security of coreboot-based platforms. Sergii Dmytruk added
+quite a few improvements related to
+[TPM support](https://review.coreboot.org/c/coreboot/+/68748),
+while Michal Zygowski added some
+[TXT improvements](https://review.coreboot.org/c/coreboot/+/59519).
+
+#### coreboot documentation improvements
+
+The work of our engineers was not closed only to changes in the code.
+It is equally important to make the code readable, clear, and understandable
+to everyone. Hence, it is worth highlighting our efforts to keep the coreboot
+documentation up to date. In this matter, the main contributors were
+[Michał Żygowski](https://review.coreboot.org/c/coreboot/+/65702) and
+[Sergii Dmytruk](https://review.coreboot.org/c/coreboot/+/68752)
+
+If you are interested in improving the security of your devices while reducing
+your dependence on proprietary firmware, you could benefit from using coreboot.
+We can also help you reduce time-to-market by simplifying the firmware
+development process. As well as if you want your firmware to be user-friendly
+and well-documented user-friendly and well-documented then coreboot is for you.
+
+---
+
+![fwupd-logo](/img/fwupd-logo.svg)
+
+[fwupd](https://3mdeb.com/shop/services/basic-firmware-update-integration/)
+is an open-source daemon that manages the firmware updates of various
+devices. You will surely benefit from using fwupd if you are interested in
+automating the firmware update process on your devices and reducing the risk of
+security vulnerabilities. fwupd is compatible with a wide range of devices,
+including laptops, desktops, and IoT devices.
+
+Changes to fwupd can be divided into several areas. The first is the addition of
+[support for Qubes OS](https://github.com/fwupd/fwupd/pull/2710/commits/295418ef784408503b6a62991a74aa4bc886c4e0).
+These changes allow firmware updates from within one of the most
+secure operating systems. Norbert Kamiński was responsible for these changes.
+
+The second significant set of changes was the addition of
+[support for FreeBSD](https://github.com/fwupd/fwupd/pull/3330). These changes
+made it possible to open fwupd to another group of operating systems.
+These changes were worked on by Michal Kopeć, Sergii Dmytruk, and Norbert Kamiński.
+
+Another sizable change that our engineers were responsible for was extending
+[flashrom support](https://github.com/fwupd/fwupd/commit/2bac03eee1a68becc96712d3a86656cc2e8fa19d)
+and adding support for Tuxedo laptops.
```

Comment:
Link to TUXEDO and maybe Dasharo Universe? How this change can help other customers?

Reply:
Link and description added https://github.com/3mdeb/news-and-ideas/pull/519/commits/68a72cdebcee326365faf61a3b7861f718765122#diff-5dff579917e1df023c3d68f093f637f7fb497ee2f22650cc786ac30a195a7eceR118